backend/src/main/java/com/htecgroup/skynest/SkyNestApplication.java

```diff
@@ -27,12 +27,12 @@ public void addCorsMappings(CorsRegistry registry) {
             .addMapping("/**")
             .allowedOrigins("http://localhost:3000")
             .allowedHeaders(
-                SecurityConstants.HEADER_STRING,
+                JwtUtils.HEADER_STRING,
```

Comment:
Since you are refactoring this :D **_HEADER_STRING_** doesn't tell me anything, rename it to something more descriptive maybe **_AUTH_HEADER_**